iroha_2_whitepaper.md

```diff
@@ -240,10 +240,17 @@ The proxy tail should at this point have received the block from at least one of
 
 **Possible faulty cases related to the leader are:**
 
-- leader ignores all transactions and never creates a block
-
-  - the solution to this is to have other nodes broadcast a transaction across the network and if someone sends a transaction to the leader and it gets ignored, then the leader can be suspected; the suspect message is sent around the network and a new leader is elected if *f*+1 nodes cannot get a reply from the leader for any transaction
-
+- Leader does not send a receipt of receiving transaction(s) from a node in the specified time (*tx receipt time*).
+  
+  - The peer broadcasts a suspect message with the tx that it tried to send to the leader. 
```

Comment:
use fully qualified name - `transaction`
Also you may need to rephrase this bullet - I didn't get the second half of it - who tried to send transaction and was a transaction suspicious or the message?